llvm/unittests/TargetParser/TargetParserTest.cpp

```diff
@@ -1331,7 +1331,10 @@ TEST(TargetParserTest, AArch64ExtensionFeatures) {
       AArch64::AEK_SVE_F16F32MM, AArch64::AEK_SVE_AES2,
       AArch64::AEK_SSVE_AES,     AArch64::AEK_F8F32MM,
       AArch64::AEK_F8F16MM,      AArch64::AEK_LSFE,
-      AArch64::AEK_FPRCVT,       AArch64::AEK_CMPBR};
+      AArch64::AEK_FPRCVT,       AArch64::AEK_CMPBR
```

Comment:
Nashe, I updated your PR using GitHub's web editor to resolve conflicts after Marian's change, but somehow missed a comma, but I can't fix this silly typo anymore:
```suggestion
      AArch64::AEK_FPRCVT,       AArch64::AEK_CMPBR,
```